etl/cris_import/utils/settings.py

```diff
@@ -8,8 +8,16 @@
     "v2_large": (2496, 3036, 4836, 5464),
 }
 
+"""This is the bbox used to crop the crash narrative"""
+NARRATIVE_BBOX_PIXELS = {
+    "v1_small": (30, 928, 681, 1590),
+    "v1_large": (296, 3683, 2580, 5749),
+    "v2_small": (30, 791, 650, 1430),
+    "v2_large": (90, 3026, 2496, 5466),
+}
```

Comment:
it occurred to me that we might want to start storing this version in the database. another option would be to use the python PDF package to actually add metadata to the PDF itself indicating which version.

i spent quite a lot of time testing different ways to identify which PDF version is being used, including by parsing the CR3 from revision date using OCR. i learned two things. 1) there are many more form revisions than what we test for (at least 5 as far as i can tell) and 2) it is hard to reliable to distinguish between them.

I could expand on this and have more thoughts on it if y'all want to chat CR3 versions. My short summary is that what we're doing now captures 99% of what we need.